docs/guides/measure-qubits.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: Measure qubits
+description: Learn how to measure qubits, including constraints on where measurements can be used.
+---
+
+# Measure qubits
+
+To get information about a qubit's state, you can _measure_ it onto a [classical bit](/api/qiskit/circuit#qiskit.circuit.Clbit). In Qiskit, measurements are performed in the computational basis, that is, the single-qubit Pauli-$Z$ basis. A measurement will thus yield 0 or 1, depending on the overlap with the Pauli-$Z$ eigenstates $|0\rangle$ and $|1\rangle$:
+
+$$
+|q\rangle \xrightarrow{measure}\begin{cases}
+      |0\rangle (\text{outcome}+1), \text{with probability } p_0=|\langle q|0\rangle|^{2}\text{,} \\
+      |1\rangle (\text{outcome}-1), \text{with probability } p_1=|\langle q|1\rangle|^{2}\text{.}
+    \end{cases}
+$$
+
+## Apply a measurement to a circuit
+
+There are several ways to apply measurement to a circuit:
+
+### `QuantumCircuit.measure` method
```

Comment:
Below you're not pre-pending `QuantumCircuit` to the methods `measure_active` and `measure_all`, should it also be removed in this title?
```suggestion
### `measure` method
```